backend/.env

```diff
@@ -4,4 +4,6 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server and MongoDB (Preview).
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="postgresql://root:root@localhost:5432/przewodnik?schema=public"
\ No newline at end of file
+DATABASE_URL="postgresql://root:root@localhost:5432/przewodnik?schema=public"
```

Comment:
Zamienmy ten plik na .example.env a .env musi byc usuniety i dodany do gitignore. 

To dlatego że nie chcemy commitować czegoś co może zostać automatycznie załadowane do środowiska potem z devowymi zmiennymi.